### PR TITLE
Add Django Debug Toolbar to local dev environments

### PR DESCRIPTION
**Pull request by suecarmol**
## Description
Added the Django Debug Toolbar library.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This library will help us with debugging issues, such as the one described in [T240673](https://phabricator.wikimedia.org/T240673)

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
 [T240673](https://phabricator.wikimedia.org/T240673)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Run `docker-compose build && docker-compose up`. This shouldn't display the Django Debug Toolbar.
2. Run `docker-compose build --build-arg REQUIREMENTS_FILE=local.txt && docker-compose up`. This should display the Django Debug Toolbar.

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
**Docker build with a REQUIREMENTS_FILE arg**
Screen Shot 2020-10-01 at 12 20 54

**Docker build without a REQUIREMENTS_FILE arg**
Screen Shot 2020-10-01 at 12 21 56

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,8 +2,12 @@
 FROM python:3.8-buster as eventstream
 
 WORKDIR /app
+ARG REQUIREMENTS_FILE
+ENV REQUIREMENTS_FILE=${REQUIREMENTS_FILE:-django.txt}
 COPY requirements/django.txt /app/
-RUN pip install -r django.txt
+COPY requirements/local.txt /app/
+RUN echo $REQUIREMENTS_FILE
+RUN pip install -r $REQUIREMENTS_FILE
```
Comment (jsnshrmn) [inline Dockerfile:10]: These can be joined in a single `RUN` command, reducing the layer count in this image.

File: Dockerfile [hunk 2]
```diff
@@ -2,8 +2,12 @@
 FROM python:3.8-buster as eventstream
 
 WORKDIR /app
+ARG REQUIREMENTS_FILE
+ENV REQUIREMENTS_FILE=${REQUIREMENTS_FILE:-django.txt}
 COPY requirements/django.txt /app/
-RUN pip install -r django.txt
+COPY requirements/local.txt /app/
```
Comment (jsnshrmn) [inline Dockerfile:8]: Both requirements files can be copied with a single `COPY` command, reducing the layer count in the image.

**Review by jsnshrmn — CHANGES_REQUESTED** [review]
This looks ready to merge once you streamline the layers in the docker image.

File: Dockerfile (hunk 1 above)
Comment (suecarmol) [reply Dockerfile]: Done!

File: Dockerfile (hunk 2 above)
Comment (suecarmol) [reply Dockerfile]: Done!

**Review by jsnshrmn — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**